processes/third_party_vuln_process.md

```diff
@@ -102,6 +102,16 @@ regular basis.
 Members of this team are required to follow the same NDA and privacy measures
 as the [Node.js Security Team](https://github.com/nodejs/security-wg/blob/master/processes/security_team_members.md).
 
+# Use of CVEs and reference
+
+Every vulnerability disclosed by the triage team through HackerOne must
+be assigned a CVE number.
+
+Vulnerabilities disclosed to this repository without going through triage
+do not need to be assigned a CVE number.
```

Comment:
I would note that we _could_ assign CVEs if there is none for a given vulnerability, if asked by the module maintainer.

Reply:
@mcollina if that good for you, I suggest we merge and update when our CNA status regarding ecosystem is set up :)

Reply:
@vdeturckheim «do not need» looks too strong, almost-forbidding here.
Perhaps reprhase «do not require assigning a CVE number, but can have it if a CVE number was assigned by any external entity» for now and update that later once things change for us?

Reply:
👍

Reply:
How about "Vulnerabilities disclosed to this repository without using HackerOne currently cannot be assigned a CVE by the triage team (we are working to resolve this) but may have a CVE number if was assigned by another entitiy.

Reply:
lgtm. updating and merging :)